ssn/ssn_separated/ssn.ttl

```diff
@@ -670,9 +658,9 @@ ssn:Observation rdf:type owl:Class ;
                                   owl:minCardinality "0"^^xsd:nonNegativeInteger
                                 ] ,
                                 [ rdf:type owl:Restriction ;
-                                  owl:onProperty ssn:featureOfInterest ;
+                                  owl:onProperty sosa:featureOfInterest ;
```

Comment:
Who would race for this ? Sounds childish...
I race for the completion of the action that was assigned to me. 
I removed isPropertyOf and hasProperty which need to be discussed in a separate thread.
Now I believe it's ready and acceptable. 
If you think otherwise, what shall I update exactly ?